krausening-python/src/krausening/logging/log_manager.py

```diff
@@ -5,16 +5,17 @@ class LogManager:
     """
     Class for handling logging.
     """
```

Comment:
I (Informational): this came from running the latest version of Habushu on the existing codebase